packages/extensions/dataflow/src/lnode/lnode.ts

```diff
@@ -25,7 +27,17 @@ export type DataAttribute = {
 export type DataAttributeSpecification = {
 	id: string
 	name: string
+	desc: string
 	dataObjectSpecificationId: string
+	subscriberLNode?: SubscriberLNode
+}
+
+export type SubscriberLNode = {
+	id: string
+	uuid: string
+	inputName: string
+	service: DataflowType | undefined
```

Comment:
Optional: just a hint, to stay consistent with other optional attributes which you defined with ? in the other objects
```suggestion
	service?: DataflowType | undefined
```